edx_prefectutils/snowflake.py

```diff
@@ -461,6 +470,16 @@ def export_snowflake_table_to_s3(
     table_name = qualified_table_name(sf_database, sf_schema, sf_table)
     export_path = os.path.join(s3_path, table_name.replace('.', '-').lower()) + '/'
 
+    parsed_path = urlparse(export_path)
+    export_bucket = parsed_path.netloc
+    export_prefix = parsed_path.path.lstrip('/')
+
+    # Snowflake's COPY INTO command's OVERWRITE option is not reliable, so we need to delete the data manually
```

Comment:
I have a question about how `OVERWRITE` works. Assume you have files A and B in your bucket and you go to run a copy into command, with OVERWRITE, but only operating with file A. Is the expectation that Snowflake's COPY INTO command would result in a bucket containing only A, or merely overwrite file A, leaving file B intact?

Reply:
The COPY command will only remove files that match the files that the COPY command unloads. 
I've seen cases where a previous run of COPY command generated files `A` and `B`, and a future run with overwrite generates files `A` and `C`, this future run would overwrite files `A` and `C` but would leave `B`, which is not what we want as they might get loaded into aurora downstream of this.

Reply:
That is why I also added the manifest generation and load from manifest option to have better control on what gets loaded into MySQL.